docs/developer-docs/integrations/bitcoin/ckbtc.md

```diff
@@ -2,37 +2,476 @@
+
+The fee can change when a new Bitcoin block is mined in the meantime, which causes the Bitcoin canister to update the Bitcoin miner fees or when another retrieval request is handled first, spending some of the outputs that were used when estimating the fee.
+
+### `get_deposit_fee`
+The endpoint returns the fee that the ckBTC minter charges for minting ckBTC when receiving new UTXOs. Currently, this fee is simply the KYT fee.
+
+### `get_withdrawal_account`
+The function returns the caller’s withdrawal account, which is the account derived from the ckBTC minter’s principal ID and the subaccount derived from the caller’s principal ID.
+
+A user can only retrieve BTC by first transferring ckBTC to this particular account.
+
+### `retrieve_btc(address: text, amount: nat64)`
+The function instructs the ckBTC minter to burn the specified amount. Once the request is picked up from the queue, a Bitcoin transaction is created containing an output that transfers the requested amount (minus fees as discussed below) to the specified Bitcoin address.
+
+Since Bitcoin retrieval is handled asynchronously, the function returns the block index of the transaction burning the ckBTC tokens.
+
+### `retrieve_btc_status(block_index: nat64)`
+The status of a BTC retrieval request can be checked using this function. The different possible statuses are:
+
+- `Unknown`: There is no information available on the ckBTC minter because there is no retrieval request associated with the given block index or the retrieval request is old and the corresponding information has already been removed.
+- `Pending`: The BTC retrieval request is queued.
+- `Signing`: The BTC retrieval request is acquiring all signatures to serve the BTC retrieval request.
+- `Sending`: The Bitcoin transaction serving the BTC retrieval request is being sent.
+- `Submitted`: The Bitcoin transaction has been transmitted. The transaction ID is returned.
+- `AmountTooLow`: The BTC retrieval request could not be served because the Bitcoin miner fees are prohibitively high.
+- `Confirmed`: The Bitcoin transaction serving the BTC retrieval request is confirmed inside the ckBTC minter, which happens when the transaction has at least the minimum required number of confirmations specified in the min_confirmations parameter above.
+
+### `get_minter_info`
+The function returns the following parameters internal to the ckBTC minter:
+
+- `kyt_fee
+- `min_confirmations
+- `retrieve_btc_min_amount
+
+### `get_events(start: nat64, length: nat64)`
+The ckBTC minter tracks the events that change its internal state. Given a start index and a length parameter, the ckBTC minter returns all events sequentially from the event at the given start index up to the event at index start+length-1.
+
+Note that this endpoint is used for debugging purposes and there is no guarantee that the endpoint will continue to exist in this form.
+
+## ckBTC minter API examples
```

Comment:
I think this section needs some work. There are actually no examples provided. Instead, each section just provides some type definitions.
Given that the API was already provided above, is the goal here to show some example invocations? This might be more useful than listing the Candid file.

Reply:
I've removed this section per our conversation on Slack.